apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -722,6 +750,90 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
         };
     };
 
+    /**
+     * Generate SCIM Bulk Request Body
+     *
+     * @param attributeMapping - attribute mapping.
+     * @returns SCIMBulkRequestBody
+     */
+    const generateMultipleUsersSCIMRequestBody = (): SCIMBulkEndpointInterface => {
+        // Create the data operations
+        const operations: SCIMBulkOperation[] = [];
+
+        // Create the user record
+        emailData?.map((email: string) => {
+            const userDetails: UserDetailsInterface = {
+                emails: [
+                    {
+                        primary: true,
+                        value: email
+                    }
+                ],
+                schemas: [
+                    "urn:ietf:params:scim:schemas:core:2.0:User",
+                    "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"
+                ],
+                userName: 
+                    userstore.toLowerCase() !== PRIMARY_USERSTORE.toLowerCase()
+                        ? `${userstore}/${email}`
+                        : email,    
+                [ userstore.toLowerCase() !== PRIMARY_USERSTORE.toLowerCase()
+                    ? UserManagementConstants.CUSTOMSCHEMA
+                    : UserManagementConstants.ENTERPRISESCHEMA
+                ]: {
+                    askPassword: "true"
+                }
+            };
+            const asyncOperationID: string = uuidv4();
+            
+            const SCIMBulkOperation: SCIMBulkOperation = {
+                bulkId: `bulkId:${email}:${asyncOperationID}`,
+                data: userDetails,
+                method: "POST",
+                path: "/Users"
+            };
+
+            operations.push(SCIMBulkOperation);
+        });
+
+        return {
+            Operations: operations,
+            failOnErrors: 0,
+            schemas: [ UserManagementConstants.BULK_REQUEST_SCHEMA ]
+        };
+    };
+
+    /**
+     * Handle multiple user invite.
+     */
+    const manualInviteMultipleUsers = async () => {
+        const scimRequestBody1: SCIMBulkEndpointInterface = generateMultipleUsersSCIMRequestBody();
```

Comment:
can we use a different variable name ?

Reply:
Updated with a14bc5214de7a4f33ef9c2c2ff4a5a3b320065d0